extensions/amp-analytics/0.1/vendors/navegg.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const NAVEGG_CONFIG = /** @type {!JsonObject} */ ({
+  'requests': {
+    'host': 'amp.navdmp.com',
+    'path': 'amp',
+    'params':
+      'aid=${clientId(navegg_id)}' +
+      '&url=${canonicalUrl}' +
+      '&ref=${documentReferrer}' +
+      '&tit=${title}' +
+      '&lan=${browserLanguage}',
+    'pageview':
+      'https://${host}/${path}?${params}&acc=${account}&v=7',
+  },
+  'triggers': {
+    'trackpageview': {
+      'on': 'visible',
+      'request': 'pageview',
+    },
+  },
+  'transport': {
+    'xhrpost': false,
```

Comment:
Want to confirm with you that your server doesn't support POST request. Asking because we recommend using `beacon` and AMP handles falling back to `xhrpost` when `beacon` is not supported in browser, and falling to `image` when `xhrpost` and `beacon` are not supported.

Reply:
Yes, the server does not accept POST request. The expected behavior is to accept only image.